unittests/Feature/FeatureModelTransaction.cpp

```diff
@@ -16,42 +16,58 @@ class FeatureModelModificationTest : public ::testing::Test,
 protected:
   void SetUp() override {
     FeatureModelBuilder B;
-    B.addParent("a", "root")->makeFeature<BinaryFeature>("a", true);
+    B.makeFeature<BinaryFeature>("a");
     FM = B.buildFeatureModel();
   }
 
   // Dummy method to fullfill the FeatureModelModification interface
-  void exec(FeatureModel &FM) override {};
+  void exec(FeatureModel &FM) override{};
```

Comment:
```suggestion
  void exec(FeatureModel &FM) override {};
```
What version of clang-format are you using?

Reply:
I updated today to `13.0.0`(latest from `llvm-project/main`) but above change is still persistent (with even more similar diffs according to changed WS rules -- will update the PR soon).

Reply:
hmm, kk